### DISCUSSION TITLE
Compatibility with MXZ-3C30NAHZ4 (outdoor), MSZ-FS12NA-U1 and 2 MSZ-GS09NA-U1 indoor units

### DISCUSSION THREAD

**Discussion by jontires** [echavet/MitsubishiCN105ESPHome#198] [General]
I've been running this for about 4 weeks on ESP8266s and getting good results with the heatpumps in HEAT mode. I'm able to monitor and control them via the embedded web servers as well as Home Assistant which I'm using to write external temperatures to the indoor units and everything is stable.
Some observations:
1. The MSZ-FS12NA-U1 indoor unit reports the value for the compressor frequency sensor, the others report 0.
2. One of the MSZ-GS09NA-U1 indoor units reports the values for the energy usage and input power sensors, the 2nd MSZ-GS09NA-U1 reports between 1 and 8 watts for input power, and the MSZ-FS12NA-U1 reports 0 for both.
3. All 3 report the same outdoor temperature.
4. Using an external temperature source seems to have improved the consistency and accuracy of maintaining the set point.
5. The MSZ-FS12NA-U1 indoor unit has the iSee feature and the status is correctly reported.
6. The MSZ-GS09NA-U1 indoor units do not have iSee and the logs contain these lines before a warning:
[20:29:20][D][Header:091]: command: (62) data length: [10]<-- header
[20:29:20][D][READ:170]: FC 62 01 30 10 09 00 00 00 00 40 00 00 00 00 00 00 00 00 00 00 14 
[20:29:20][D][chkSum:077]: OK-> 14=14 
[20:29:20][D][CYCLE:364]: 5b: Receiving Power/Standby response
[20:29:20][D][Decoder:148]: [0x09 is sub modes]
[20:29:20][W][lookup:204]: auto mode sub mode caution: value 64 not found, returning value at index 0
So perhaps the 0x40 byte from the heatpump is saying iSee is unsupported.

I don't know if the installers did anything to designate roles among the indoor units, but I've power cycled the breaker that feeds them a few times and so far the behavior is consistent.

I really appreciate all the work that's gone into this and the supporting projects, having access to real time operational data has helped me to reduce energy usage by reducing the time the spent in defrost and preheat modes.

**Comment by luiggi78** [comment]
Hello, I'm wondering how having all this data enabled you to reduce your energy usage? I have a similar system than yours and I can tell that it is short cycling a lot (I see that on my Emporia energy monitor).  I seems that my system was oversized so I'm looking at ways to improve it. Thanks!
